src/code/z_elf_message.c

```diff
@@ -1,3 +1,62 @@
 #include "global.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_elf_message/func_800F05C0.s")
+u16 ElfMessage_GetFirstCycleHint(GlobalContext* globalCtx) {
+    if (INV_CONTENT(ITEM_OCARINA) == ITEM_OCARINA) {
+        return 0;
+    }
+    if (CURRENT_DAY <= 0) {
```

Comment:
I think it's funny this ask for `<= 0` because `CURRENT_DAY` can never be negative because of the modulo 5 operation it performs :laughing: